charts/kong/values.yaml

```diff
@@ -128,6 +128,40 @@ status:
     containerPort: 8543
 
 # Specify Kong proxy service and listener configuration
+cluster:
+  # If you want to specify annotations for the Manager service, uncomment the following
+  # line, add additional or adjust as needed, and remove the curly braces after 'annotations:'.
+  annotations: {}
+  #  service.beta.kubernetes.io/aws-load-balancer-proxy-protocol: "*"
+
+  tls:
+    enabled: false
+    servicePort: 8005
+    containerPort: 8005
+    # Set a nodePort which is available if service type is NodePort
+    # nodePort: 32443
+
+  type: NodePort
+
+  # Kong cluster ingress settings.
+  # TODO: have left this in for now, but there is no corresponding template, and it doesn't work
+  # unsure if it makes sense to have a standard ingress setup, since it's of questionable use and
+  # the hybrid mTLS presumably cannot work through an ingress proxy without considerable work
+  ingress:
```

Comment:
I initially didn't because I remembered the client TLS stuff, then forgot that and re-added it, and have now been reminded to remove it again. Explanatory docs added also.